### ISSUE TITLE
[MU4 Issue] Accessibilty: NVDA reads names of instrument families and instruments as "not selected"

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#12852]
**Describe the bug**
In the Choose instruments portion of the instruments dialog when creating a new score, NVDA reads "not selected" for each instrument or family as you browser the first two columns.  This is misleading and confusing to blind users, since they are not actually selectable.

**To Reproduce**
1. Start NVDA
2. Create new score
3. Choose instruments
4. Start browsing the instrument family and instrument lists in the first two columns

**Result**
NVDA reads "Woodwinds not selected", and if you then tab over and tart browsing the instrument list, it reads "Flute not selected"

**Expected behavior**
It should read just "Woodwinds" and "Flute", since these are not checkboxes and don't have selectable state.

**Platform information**
 - OS: Windows

**Additional context**
Orca does not read this way, not sure what, since it does read selected state for other elements.  Could be NVDA is just confused, but still, we should try to find a way to not confuse it.

**Comment by HemantAntony** [comment]
@MarcSabatella Isn't this the same issue as #15521?

**Comment by MarcSabatella** [comment]
Indeed, I guess I couldn't find this one for some reason when I submitted the other.

Another observation: it reads "not selected" and even when it is in fact selected.  For instance, in the instrument list, the "Add selected instruments to score" button obviously is operating on some concept of an instrument being selected, but if you select an instrument in the list, it *still* says not selected.

Also, in the first column, you need to select (or activate, or whatever one might need to call it) a family before its contents are loaded into the second column.  But this also is not read.

And the *names* of the columns are never read.  This is especially confusing for the second and third column, since nothing about the elements at the top of the column is obviously relevant.  When someone reaches the column, it isn't obvious what the column is actually for.

If it makes sense to close this issue in favor of the other, I can copy this comment over.

**Comment by DmitryArefiev** [comment]
Duplicate of #15521
